File: TaskManager/Lab_final_210042166/Task_Management_System/Logger_Class/Logger.java
package Lab_final_210042166.Task_Management_System.Logger_Class;

import org.mockito.internal.matchers.Null;

import java.util.ArrayList;
import java.util.List;

public class Logger {
    private static Logger instance;
    private List <String>logs;

    private Logger(){
        logs=new ArrayList<>();
    }
    public static Logger getInstance()
    {
        if (instance== null)
        {
            instance=new Logger();
        }
        return instance;
    }
    public void log(String message)
    {
        logs.add(message);
    }
    public List<String>getLogs()
    {
        return logs;
    }


}

File: TaskManager/Logger.cs
using System;
using System.Collections.Generic;

public class Logger
{
    private List<string> logs = new();

    public void Log(string message)
    {
        string logEntry = $"[{DateTime.Now:yyyy-MM-dd HH:mm:ss}] {message}";
        logs.Add(logEntry);
    }

    public List<string> GetLogs() => logs;
}